cli/cmdlineparser.cpp

```diff
@@ -503,14 +503,23 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
 
             // --project
             else if (std::strncmp(argv[i], "--project=", 10) == 0) {
-                _settings->project.import(argv[i]+10);
-                if (std::strstr(argv[i], ".sln") || std::strstr(argv[i], ".vcxproj")) {
+                const std::string projectFile = argv[i]+10;
+                const ImportProject::Type projType = _settings->project.import(projectFile);
+                if (projType == ImportProject::VS_SLN || projType == ImportProject::VS_VCXPROJ) {
                     if (!CppCheckExecutor::tryLoadLibrary(_settings->library, argv[0], "windows.cfg")) {
                         // This shouldn't happen normally.
                         printMessage("cppcheck: Failed to load 'windows.cfg'. Your Cppcheck installation is broken. Please re-install.");
                         return false;
                     }
                 }
+                if (projType == ImportProject::MISSING) {
+                    printMessage("cppcheck: Failed to load project '" + projectFile + "'. File cannot be read.");
```

Comment:
To me this message says that there was some problem while Cppcheck were reading the file. How about: `cppcheck: Failed to open file 'abc'`